runtime/common/src/lib.rs

```diff
@@ -321,6 +321,11 @@ pub enum ProxyType {
 	Swap,
 	Loan,
 	DexLiquidity,
+	StableAssetSwap,
+	StableAssetMint,
+	StableAssetRedeemProportion,
+	StableAssetRedeemSingle,
+	StableAssetRedeemMulti,
```

Comment:
they should be merged to `StableAssetLiquidity`. there is no reason to be that specific